lib/src/io.dart

```diff
@@ -569,7 +569,9 @@ final String dartRepoRoot = (() {
 
 /// A line-by-line stream of standard input.
 final Stream<String> _stdinLines =
-    ByteStream(stdin).toStringStream().transform(const LineSplitter());
+    ByteStream(stdin).toStringStream().transform(const LineSplitter()).map((x) {
+  return x;
+});
```

Comment:
Why?

Reply:
That was debug code that got incorrectly reverted.

Removed.